interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
```

Comment:
This is the only script that "changes state" (ie. can change data in database), so only place where need to use the CSRF prevention mechanism. This can be done in 3 lines in 3 separate places, so will show that in subsequent comments.